crd/mysqlcr.go

```diff
@@ -0,0 +1,13 @@
+package crd
+
+import (
+	"bytes"
+	"os"
+
+	"github.com/grtl/mysql-operator/util"
+)
+
+func MySQLcreateCrd() (bytes.Buffer, error) {
```

Comment:
Name should be CamelCase and should match custom resource type `MySQLClusterCreateCRD`

Reply:
Returning `bytes.Buffer` is not a good idea it should cover all logic for registering CRD